BenadrylAutomation/src/main/java/com/jnj/pageobjects/SignUpPage.java

```diff
@@ -30,25 +31,22 @@ public SignUpPage() {
 	 * Locators
 	 */
 	@FindBy(xpath = "//div[@class='sfmc-careclub-logo']")
-	private WebElement logo;
+	WebElement logo;
```

Comment:
All private declarations are removed ?
@RashiTiwari21

Reply:
All the locators added as private @goutamnaik007